docs/00-deployment.md

```diff
@@ -1,74 +1,156 @@
+
+#### Update the Redirect URI
+
+We need to ensure the Redirect URI configured for the application is correct - it must end with `/signin-oidc`.
+
+ 1. Select *Authentication* from the sidebar
+ 2. If needed, modify the existing reply URL to append '/signin-oidc'
+    
+    - The new URL should look like:  
+    `https://[MyWebAppName].azurewebsites.net/signin-oidc`
+
+![Redirect URI](images/00-redirect-uri.png)
+
+ 3. Press *Save* 
 
 #### Set Application Access Permissions
 
 The following instructions allow you to restrict access to specific users in your organization.
 
- 1. In the Azure portal navigate to Azure Active Directory -> Enterprise Applications
- 2. Search for the AAD application you created above using the Application ID
- 3. Click on Properties and click Yes for User Assignment Required
- 4. Click Save
- 5. Click Users and Groups
- 6. Add each user that requires access to the portal
+ 1. In the Azure portal navigate to *Azure Active Directory* -> *Enterprise Applications*
+ 2. Search for the AAD Application you created above using the Application ID noted earlier
+
+![AAD Enterprise Applications](images/00-enterprise-applications.png)
+
+ 3. Press the name of the application to open it
+ 4. Select on *Properties* and turn on *User Assignment Required*
+
+![User assignment required](images/00-user-assignment-required.png)
+
+ 5. Press *Save*
+
+
+ 6. Select *Users and Groups* from the sidebar
+ 7. Add each user or group that requires access to the portal
+
+When you are finished, all the users with access to your Azure Render Hub will be listed.
+
+![Users and Groups](images/00-assign-users.png)
 
 #### Get the AAD Tenant ID
 
-In the Azure Portal navigate to Azure Active Directory -> Properties.  Note down the Directory ID, this is your Tenant ID that is required when you deploy the portal.
+In the Azure Portal navigate to *Azure Active Directory* -> *Properties*.
+
+![AAD Properties](images/00-properties-aad-directory-id.png)
 
-### Using Cloud Shell
+Note down the Directory ID, this is your Tenant ID that will be required when you deploy the portal.
 
-Simply copy the script snippet below, update the $webAppName variable and paste the script below into the cloud shell to create a new AAD application.  Keep in mind the Web App name must be globally unique and be a valid DNS name as it becomes the host in your website's URL, e.g. https://< webAppName >.azurewebsites.net.
+### Create an Azure AD Application using Cloud Shell 
+
+*You don't need to do this if you've used the steps above to create the AAD application using the portal.*
+
+Simply copy the script snippet below, update the $webAppName variable and paste the script below into the cloud shell to create a new AAD application.  Keep in mind the Web App name must be globally unique, no longer than 20 characters, and be a valid DNS name as it becomes the host in your website's URL, e.g. `https://<webAppName>.azurewebsites.net`.
```

Comment:
add instructions on how to also create a client secret as well as how to pull out all the ID's etc that you need in the next step without going to the portal to get them.

Reply:
I'll have to review how the Cloud Shell bit works ...